lib/styles/tokens.css

```diff
@@ -167,3 +169,22 @@
   --vds-line-height-heading-lg:   48px;
   --vds-line-height-heading-xl:   54px;
 }
+
+
+/**
+ * @tokens Container Widths
+ * @presenter ContainerWidth
```

Comment:
FYI these are limited to this list of [presenters](https://github.com/UX-and-I/storybook-design-token#presenters).